staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4031,10 +4037,23 @@ type PodStatus struct {
 	// +optional
 	NominatedNodeName string `json:"nominatedNodeName,omitempty" protobuf:"bytes,11,opt,name=nominatedNodeName"`
 
-	// IP address of the host to which the pod is assigned. Empty if not yet scheduled.
+	// hostIP holds the IP address of the host to which the pod is assigned. Empty if the pod has not started yet.
+	// A pod can be assigned to a node that has a problem in kubelet which in turns mean that HostIP will
+	// not be updated even if there is a node is assigned to pod
```

Comment:
I think that this only can happen if the node has been deleted or the kublet has lost connection to the apiserver, this comment is a bit confusing

Reply:
I have no hard opinion on it, this I remember is based on the last comment to change.